tests/ecosystem/upgrade/test_noobaa.py

```diff
@@ -157,7 +158,7 @@ def test_noobaa_postupgrade(
         LOCAL_TEMP_PATH,
         mcg_obj_session
     )
-    assert bucket.phase == constants.STATUS_BOUND
+    assert bucket.status == constants.STATUS_BOUND
```

Comment:
plz add assert message (if assert fails)

Reply:
> plz add assert message (if assert fails)

My personal opinion is that assert message is not necessary in such clear cases, and don't understand the urge to add them everywhere. That said, adding a logging above the assert could help.